.github/workflows/remove-cargo-target-weeklly.yml

```diff
@@ -0,0 +1,61 @@
+name: Remove Cargo target weekly
+
+on:
+  schedule:
+    # Every day at 1 am
```

Comment:
The title is `name: Remove Cargo target weekly`, but this comment is `# Every day at 1 am`. 😕 

How about merge this into `.github/workflows/remove-old-artifacts.yml`?

Reply:
done